app/helpers/pitches_helper.rb

```diff
@@ -12,4 +12,11 @@ def open_round_2_button
 		to_close_round = Round.find_by(round_number: 1)
 		to_close_round.update(open: false)
 	end
+
+	def close_rounds_button
+		round_1 = Round.find_by(round_number: 1)
```

Comment:
This can be refactored.
```
def close_rounds_button(round_params)
    round_to_close = Round.find_by(round_params)
    round_to_close.update(open: false)
end
```